drivers/wifi/nrf700x/src/wifi_mgmt.c

```diff
@@ -948,3 +948,63 @@ int nrf_wifi_filter(const struct device *dev,
 	return ret;
 }
 #endif /* CONFIG_NRF700X_RAW_DATA_RX || CONFIG_NRF700X_PROMISC_DATA_RX */
+
+int nrf_wifi_set_rts(const struct device *dev,
+		     int rts_threshold)
+{
+	enum nrf_wifi_status status = NRF_WIFI_STATUS_FAIL;
+	struct nrf_wifi_ctx_zep *rpu_ctx_zep = NULL;
+	struct nrf_wifi_vif_ctx_zep *vif_ctx_zep = NULL;
+	struct nrf_wifi_umac_set_wiphy_info wiphy_info;
+	int ret = -1;
+
+	if (!dev) {
+		LOG_ERR("%s: dev or params is NULL", __func__);
+		return ret;
+	}
+
+	if ((rts_threshold < -1) || (rts_threshold > 65536)) {
```

Comment:
Shouldn't `65536` be `2437` ? Also pleasea use macros from `wifi.h` rather than magic numbers.